### ISSUE TITLE
Data Collection: how well will our model hold up to less data.

### ISSUE THREAD

**Issue by JosephCarey-cpp** [JosephCarey-cpp/gssiProgram#8]
Tasks 1 & 2 due 08/03 before 11:30 AM

After experimenting with 80% of samples used for training, we found the best hyperparameters and accuracy in classifying 3 or 2 families. We want now to reduce the ratio of training samples (down to 1%, or less). The goal is to understand how much the accuracy goes down when the availability of training samples is reduced.

When completing this task, we want the data that was best determined from #2 to go from 0.8 then 0.5, then 0.2, then 0.1, then 0.09 - 0.01. The line being edited is:
`train_samples_count = int(0.8 * samples_count)`
where 0.8 is the value we are changing. This line is in the preprocessing section of the TensorFlow program.

Tasks:
- [x] Two families (check mark task when complete)
- [x] Three families (check mark task when complete)

The previous spreadsheet in #2 can be used to include the data.
